kubernetes/helm_charts/core/ml-core-service/values.j2

```diff
@@ -0,0 +1,27 @@
+### Default variable file for cert-service ###
+
+namespace: {{ namespace }}
+imagepullsecrets: {{ imagepullsecrets }}
+dockerhub: {{ dockerhub }}
+
+replicaCount: {{kendra_replicacount|default(1)}}
+repository: {{kendra_repository|default('ml-core-service')}}
+image_tag: {{ image_tag }}
+resources:
+  requests:
+    cpu: {{kendra_cpu_req|default('100m')}}
```

Comment:
Can you please rename the kendra variables?

Reply:
resloved